pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
That would also allow you to extend with stuff like `/namespaces/<namespace>/metrics/services` to aggregate metrics from service endpoint pods.